src/cli/commands/config.js

```diff
@@ -16,8 +16,15 @@ export const {run, setFlags} = buildSubCommands('config', {
       return false;
     }
 
-    const [key, val = true] = args;
+    const [key, val] = ((key, val = true) => {
+      if (typeof val === 'string' && (/^(true|false)$/i).test(val)) {
+        val = (val.toLowerCase() === 'true');
+      }
+
+      return [key, val];
+    })(...args);
```

Comment:
Not a fan of the IIFE and spreading the arguments into it, we should do something like:

```javascript
const key = args[0];
let val = args[1];

if (key === 'true' ...) {
   val = val === 'true';
}
```

We don't really need to handle the alternate casing of `true`.